api/src/controllers/lottery.controller.ts

```diff
@@ -118,9 +118,22 @@ export class LotteryController {
     );
   }
 
+  @Put('autoPublishResults')
+  @ApiOperation({
+    summary: 'Trigger the lottery auto publish process job',
+    operationId: 'autoPublishResults',
+  })
+  @ApiOkResponse({ type: SuccessDTO })
+  @PermissionAction(permissionActions.update)
+  @UseInterceptors(ActivityLogInterceptor)
+  @UseGuards(OptionalAuthGuard, AdminOrJurisdictionalAdminGuard)
```

Comment:
Can't remember a ton of context on this but sort of suprised to see admin or juris here when admin don't release to public. Am I missing something on how we permission cron jobs?

Reply:
This is for the endpoint if we needed to call the cron job outside of scheduled time. All our cron jobs have this permissioning.